packages/react-magma-dom/src/components/Stepper/Step.tsx

```diff
@@ -199,68 +199,66 @@ const StyledSecondaryLabel = typedStyled.span<{
 
-    return (
-      <StyledStep {...rest} ref={ref} data-testid={testId}>
-        <StyledStepIndicator
-          hasError={hasError}
-          isInverse={isInverse}
-          stepStatus={stepStatus}
-          theme={theme}
-        >
-          {stepStatus === StepStatus.complete && !hasError && (
-            <CheckIcon aria-hidden="true" />
-          )}
-          {hasError && <CrossIcon aria-hidden="true" />}
-        </StyledStepIndicator>
+  return (
+    <StyledStep {...rest} ref={ref} data-testid={testId}>
+      <StyledStepIndicator
+        hasError={hasError}
+        isInverse={isInverse}
+        stepStatus={stepStatus}
+        theme={theme}
+      >
+        {stepStatus === StepStatus.complete && !hasError && (
+          <CheckIcon aria-hidden="true" />
+        )}
+        {hasError && <CrossIcon aria-hidden="true" />}
+      </StyledStepIndicator>
 
-        <StyledStepTextWrapper>
-          {!areLabelsHidden ? (
-            <>
-              {label && (
-                <StyledLabel
-                  label={label}
-                  isInverse={isInverse}
-                  data-testid={testId && `${testId}-label`}
-                  theme={theme}
-                >
-                  {label}
-                </StyledLabel>
-              )}
-              {secondaryLabel && (
-                <StyledSecondaryLabel
-                  secondaryLabel={secondaryLabel}
-                  isInverse={isInverse}
-                  data-testid={testId && `${testId}-secondaryLabel`}
-                  theme={theme}
-                >
-                  {secondaryLabel}
-                </StyledSecondaryLabel>
-              )}
-            </>
-          ) : (
-            <HiddenLabelText>
-              {label && secondaryLabel
-                ? `${label} ${secondaryLabel}`
-                : label || secondaryLabel}
-            </HiddenLabelText>
-          )}
-        </StyledStepTextWrapper>
-      </StyledStep>
-    );
-  }
-);
+      <StyledStepTextWrapper>
+        {!areLabelsHidden ? (
+          <>
+            {label && (
+              <StyledLabel
+                label={label}
+                isInverse={isInverse}
+                data-testid={testId && `${testId}-label`}
+                theme={theme}
+              >
+                {label}
+              </StyledLabel>
+            )}
+            {secondaryLabel && (
+              <StyledSecondaryLabel
+                secondaryLabel={secondaryLabel}
+                isInverse={isInverse}
+                data-testid={testId && `${testId}-secondaryLabel`}
+                theme={theme}
+              >
+                {secondaryLabel}
+              </StyledSecondaryLabel>
+            )}
+          </>
+        ) : (
+          <HiddenLabelText data-testid="hello again">
+            {label && secondaryLabel
+              ? `${label} ${secondaryLabel}, ${stepStatus}`
+              : label || secondaryLabel}
```

Comment:
Maybe this is a question for Orion, but how _do_ we want this announced? I can't find any examples of this.

In the example below, it would say "Choose materials, complete" -> "Additional details, active" -> "Confirm, incomplete". 
<img width="291" alt="image" src="https://github.com/cengage/react-magma/assets/91160746/a21d17bc-4065-4ef7-ab5f-a375f82b36e9">

Reply:
Based on the feedback from Perkins, we only have to specify the complete steps with hidden text. The aria attributes will tell the screen reader which step they're currently on. But they did call out that we're not communicating the number of steps to screen readers. So I think it would be more like:

- Step 1 of 4. Choose materials. Step completed.
- Step 2 of 4. Additional details  `aria-current="step"`
- Step 3 of 4. Confirm.
- Step 4 of 4. Finish.